src/lemlib/movements/purepursuit.cpp

```diff
@@ -101,22 +102,22 @@ std::pair<int, int> PurePursuit::update(Pose pose) {
         Pose intersect = circleLineIntersect(last, cur, pose, lookaheadDist);
         // if the intersection is not the robot's current position, then we have found the lookahead point
         if (intersect != pose) {
-            lookahead = {intersect.x, intersect.y, 0, 0, i};
+            lookahead = {intersect.x, intersect.y, 0_deg, 0_rpm, i};
```

Comment:
```suggestion
            lookahead = {intersect.x, intersect.y, 0_rad, 0_inps, i};
```
Change to correct units